bref/symfony-messenger-sns/0.2/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-package": {
+        "config/": "%CONFIG_DIR%/packages/"
+    },
+    "env": {
+        "AWS_SNS_DSN": "enqueue://acme?topic[name]=my_sns_topic",
```

Comment:
Where does `acme` come from? Can't we use a real default value?

Reply:
"acme" is a reference to the php-enqueue config value. 

I could change it do `default` to match the recipe: https://github.com/symfony/recipes-contrib/blob/master/enqueue/enqueue-bundle/0.9/config/packages/enqueue.yaml